src/commands/close.js

```diff
@@ -0,0 +1,27 @@
+const getConfig = require('../config/get-config.js');
+const config = getConfig();
+module.exports = {
+  prefix: 'close',
+  description: 'Closes the channel.',
+  command: async function (message) {
+    try {
+      const target = message.channel;
+      //check for log channel
+      const log = message.guild.channels.cache.find(
+        (channel) => channel.name === config.LOG_MSG_CHANNEL
+      );
+      //check for user permissions
+      if (!message.member.hasPermission('MANAGE_CHANNELS')) {
+        console.log('Missing permissions.');
+      }
```

Comment:
### This doesn't seem to do anything if the user doesn't have permissions to manage channels allowing anyone to close any channel by calling this command.

Reply:
Oops! Forgot the return statement 😅

Reply:
This is corrected